plugins/rsyslog.yaml

```diff
@@ -47,6 +47,42 @@ parameters:
     default: ""
     advanced_config: true
     hidden: true
+  - name: tls_enable
+    label: "TLS Configuration Enable"
+    description: Should TLS be used? Defaults to false.
+    type: boolean
+    default: false
+    advanced_config: true
+    relevant_if:
+      connection_type:
+        equals: "tcp"
+  - name: tls_certificate
+    label: "TLS Configuration Certificate"
+    description: Path to the certificate or certificate chain to use for TLS
+    type: string
+    default: ""
+    advanced_config: true
+    relevant_if:
+      connection_type:
+        equals: "tcp"
+  - name: tls_private_key
+    label: "TLS Configuration Private Key"
+    description: Path to the certificates private key to use for TLS
```

Comment:
Mention required format.

Reply:
@Dylan-M want to make sure this is not missed. We should specify certificate format requirements for the private key and certificate.